clients/cli/devices.c

```diff
@@ -2709,55 +2893,33 @@ do_device_wifi_list (NmCli *nmc, int argc, char **argv)
 	if (argc)
 		g_printerr (_("Unknown parameter: %s\n"), *argv);
 
-	if (ifname) {
+	if (rescan == NULL || strcmp (rescan, "auto") == 0) {
+		struct timespec tp;
+
+		if (clock_gettime (CLOCK_BOOTTIME, &tp) == -1) {
```

Comment:
NetworkManager falls back to CLOCK_MONOTONIC, if CLOCK_BOOTTIME is not supported. Client-side we should behave identically. Possibly there should be a public API function in libnm which returns a suitable "now" timestamp.

Reply:
CLOCK_BOOTTIME is there since 2.6.39, we can safely assume it's always there.
The CLOCK_MONOTONIC is there just so that unit tests succeed on RHEL-6 kernel.